README.md

```diff
@@ -19,7 +30,166 @@ After installation, you will need to install `react-navigation` and its peer dep
 
 ## Examples
 
-- example built-in library - [/example](/example)
+- example built-in library - check [/example](/example)
+
+## API
+
+For all navigator you need follow some steps:
+
+### Config
+
+First of all, create a config file like `Config.re` and there define your all routes. It sould be a simple Varian Type with your routes/tabs/items
+
+```ReasonML
+type route =
+  | Home
+  | UserDetails;
+```
+
+> It is important to create a separate file for that because ReasonML is aware of circular dependencies.
```

Comment:
Would replace "for that because ReasonML is aware of circular dependencies." with something easier to understand. 

Maybe: "in order to avoid circular dependencies when you try to import navigation"

Reply:
yep, that make more sense ;)